spinalcordtoolbox/deepseg/models.py

```diff
@@ -87,10 +91,14 @@
         "contrasts": ["mp2rage"],
         "default": False,
     },
-    "model_seg_ms_lesion_mp2rage": {
-        "url": [
-            "https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20211223/model_seg_ms_lesion_mp2rage.zip"
-        ],
+    "model_seg_lesion_mp2rage": {
+        "url": {
+            "seed7": ["https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20230210//model_seg_lesion_mp2rage_seed7.zip"],
+            "seed8": ["https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20230210//model_seg_lesion_mp2rage_seed8.zip"],
+            "seed9": ["https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20230210//model_seg_lesion_mp2rage_seed9.zip"],
+            "seed10": ["https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20230210//model_seg_lesion_mp2rage_seed10.zip"],
+            "seed11": ["https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20230210//model_seg_lesion_mp2rage_seed11.zip"],
+        },
```

Comment:
Release `r20230210` has two main issues that need to be resolved before merging:

- [ ] https://github.com/ivadomed/model_seg_ms_mp2rage/issues/69
- [ ] https://github.com/ivadomed/model_seg_ms_mp2rage/issues/70

Reply:
Fixed in https://github.com/ivadomed/model_seg_ms_mp2rage/releases/tag/r20230210